examples/demo/updating_plot/updating_plot2.py

```diff
@@ -96,7 +103,7 @@ def _create_window(self):
 
         self.timer = Timer(100.0, self.onTimer)
         self.container = container
-        return Window(self, -1, component=container)
+        return container
 
     def onTimer(self, *args):
         for plot in self.animated_plots:
```

Comment:
this example (and other `updating_plot*.py` examples) work as expected - but the window sizes are too small. From the looks of it, the `size` argument passed to `demo_main` isn't used so simply asking for a larger size there does not help.

Reply:
yeah not sure why that exists anymore, it can probably be removed in enable. 

But also, Should we even be importing from `enable.example_support` in the first place

Reply:
> But also, Should we even be importing from enable.example_support in the first place

I would say no. It's a small wrapper (`demo_main`/`DemoFrame`) that adds an unnecessary abstraction in the middle IMO, especially in a user-facing example. But let's discuss and handle that separately.

Reply:
I've opened #715